### PR TITLE
Use /examples as source of truth for example-driven test cases

### PR DESCRIPTION
**Pull request by Jberlinsky**
See https://github.com/terraform-google-modules/terraform-google-container-vm/pull/3#issuecomment-430937734 for discussion

### CODE REVIEW

**Comment by morgante** [comment]
README instructions for running tests also needs to be updated: https://github.com/terraform-google-modules/terraform-google-container-vm#terraform-integration-tests

Also I'd like to see a single terraform.tfvars for all tests (you can symlink), or push people to use env variables, to reduce the duplication needed. Copy and pasting 3 different files with identical values is tedious.

**Comment by Jberlinsky** [comment]
@morgante The solution we discussed is now in this PR.

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
-test/**/terraform.tfvars
-test/**/.terraform
-test/**/terraform.tfstate.d
-test/**/ssh/key
+examples/**/terraform.tfvars
+examples/**/.terraform
+examples/**/terraform.tfstate.d
+examples/**/ssh/key
 *.pyc
+test/fixtures/terraform.tfvars
```
Comment (morgante) [inline .gitignore]: I'd suggest locating this in the examples directory instead.

Comment (Jberlinsky) [reply .gitignore]: @morgante Might I ask why? The variables file seems more relevant as a test-related fixture to me.

Comment (morgante) [reply .gitignore]: You also need to fill in the variables to run the examples too.

**Pull request merged**